lib/IRGen/GenExistential.cpp

```diff
@@ -1407,7 +1443,27 @@ static const TypeInfo *createExistentialTypeInfo(IRGenModule &IGM, CanType T) {
   OpaqueExistentialLayout opaque(protosWithWitnessTables.size());
   Alignment align = opaque.getAlignment(IGM);
   Size size = opaque.getSize(IGM);
+  // There are spare bits in the metadata pointer and witness table pointers
+  // consistent with a native object reference.
+  SpareBitVector spareBits;
+  spareBits.appendClearBits(size.getValueInBits());
+  /* TODO: There are spare bits we could theoretically use in the type metadata
```

Comment:
Don't forget to clean this up.

Reply:
I had left this intentionally, for reference in case we ever work around this restriction. Technically even opaque existentials are always bitwise-takable now, so they *could* be made loadable in theory, allowing us to manipulate their spare bits if we ever made that change.